roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -11,14 +11,12 @@
     "volumes": [
         "/var/lib/openstack/config/nova:/var/lib/kolla/config_files:ro",
 {% if edpm_nova_tls_certs_enabled %}
-	"/var/lib/openstack/certs/nova/tls.crt:/etc/pki/nova/server-cert.pem:ro",
-	"/var/lib/openstack/certs/nova/tls.key:/etc/pki/nova/private/server-key.pem:ro",
-	"/var/lib/openstack/cacerts/nova/tls-ca-bundle.pem:/etc/pki/CA/cacert.pem:ro",
+	"{{ edpm_nova_tls_ca_src_dir }}/tls-ca-bundle.pem:/etc/pki/CA/cacert.pem:ro",
 {% endif %}
         "/etc/localtime:/etc/localtime:ro",
         "/lib/modules:/lib/modules:ro",
         "/dev:/dev",
-        "/run/openvswitch:/run/openvswitch",
+        "/run/openvswitch:/run/openvswitch:shared",
```

Comment:
fyi the only reason i added shared is for parity with libvirt and incase for some reason the nova_compute container stated before ovs on the host

in that case shared should allow nova to see the ovsdb socket when it gets created on the host.